doc/sfsmount.1.adoc

```diff
@@ -213,7 +213,7 @@ Equivalent to *--nostdopts* (*-n*) option for use in fstab.
 *-o nonempty*::
 Equivalent to *--nonempty* option for use in fstab.
 
-*-o sfsignoreflush*::
+*-o sfsignoreflush*'0|1'::
 Advanced: use with caution. Ignore flush usual behavior by replying SUCCESS to it
 immediately. Targets fast creation of small files, but may cause data loss
 during crashes when allegedly flushed data is still being processed.
```

Comment:
Please also add the default value

Reply:
Done. And about adding a log when activated, I considered better to let it as it is right now because this option can be changed on runtime, and this way we keep consistency and simplicity.